Core/include/Acts/EventData/Charge.hpp

```diff
@@ -128,7 +128,7 @@ struct SinglyCharged {
   /// @param qOverP Charge over momentum
   /// @return Signed elementary charge (+e or -e)
   constexpr float extractCharge(double qOverP) const noexcept {
-    return std::copysign(UnitConstants::e, qOverP);
+    return copySign(UnitConstants::e, qOverP);
```

Comment:
looking at this again I think this and similar cases below do not really benefit from the new `copySign` and should rather use `std::copysign`. where we depend on the vector in/out `copySign` makes sense